Documentation/demo.md

```diff
@@ -0,0 +1,286 @@
+### Demo
```

Comment:
heading hierarchy: this should be `#`, the headings below should all decrement by at least 2 hashes